packages/core/src/new-api/types/journal.ts

```diff
@@ -225,7 +225,7 @@ export type OnchainResultMessage =
   | OnchainResultSuccessMessage
   | OnchainResultFailureMessage;
 
-export type OnchainResultSuccessMessage =
+type OnchainResultSuccessMessage =
```

Comment:
Why remove the export? I think this contradicts the api-extractor recommendations: if a type is used in an exported type, it should also be exported.

Reply:
At least for public types